vmm/src/lib.rs

```diff
@@ -2101,14 +2114,8 @@ impl Vmm {
 
     fn configure_from_json(
```

Comment:
It would would be better to rename this function since it does not deal with JSON anymore.

Reply:
Renamed it.